### ISSUE TITLE
Known Chrome extension broken ?

### ISSUE THREAD

**Issue by benborges** [mapkyca/KnownChrome#4]
concerns latest Known only. older Known extension install are working just fine

When i click the big K icon on Chrome i get the popup with Share, Reply, Bookmark, RSVP
until here it's all fine.

when you switch to Reply or Bookmark or RSVP, the little window popup refresh but always come back to Share, there is no way to select the type of post.

if necessary i can do a short video of what I see on my side, so that it could help better understand
the issue?

so in short if you use it to share a status, it works fine. but there is no way to switch between the type. 

ps: also it seems the Bit.ly plugin for chrome mess up with Chrome extension when activated. 
it seems to want to create a shortened URL when you open the K Chrome extension and display
a yellow notice that it can't shorten the url. (not sure why that plugin wants to shorten a URL at this point when there isn't even a URL to shorted yet)

**Comment by mapkyca** [comment]
<a href="" class="u-in-reply-to"></a><a href="https://github.com/mapkyca/KnownChrome/issues/4" class="u-in-reply-to"></a>Hmm... I've not been able to replicate this (using the Chrome plugin on the latest codebase to reply to this ticket as it happens!). What other plugins do you have running? Perhaps there's something in your plugin/theme that's interfering? 

Regarding Bit.ly, assuming it's the plugin I wrote, I'm also using this with no problem.

Due to the way the share page works, it tries to shorten the page url automatically (for sharing) even if you want to reply. To get it working you need to link to you account, either by static key or OAuth2 (via your admin & settings pages).

**Comment by benborges** [comment]
It was PHP PHEAR not installed it seems, since this got fixed, i can install the generated crx and install the app. only issue arise when one update Chrome or Chromium, at first appplication restart the Known chrome app get deactivated and need to be re installed. beside that, it's working great !

**Comment by mapkyca** [comment]
Glad to hear it!

Odd about the update - I'm not getting that deactivation, mind you I update chrom(e|ium) through my distro's package manager, so maybe it works a differently..
